pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -0,0 +1,189 @@
+            pokemon['spawn_point_id'] = pokemon['spawnpoint_id']
+            pokemon['disappear_time'] = int(pokemon['disappear_time'] / 1000)
+            pokemon['name'] = self.pokemon_data[pokemon['pokemon_id'] - 1]['Name']
+            pokemon['is_vip'] = pokemon['name'] in self.bot.config.vips
+
+            if pokemon['name'] not in self.config['catch'] and pokemon['name'] not in self.bot.config.vips:
+                continue
+
+            if pokemon['disappear_time'] < (now + self.config['min_time']):
+                continue
+
+            if self.wasCaught(pokemon):
+                continue
+
+            if pokemon['name'] in self.config['catch']:
+                pokemon['priority'] = self.config['catch'][pokemon['name']]
+            else:
+                pokemon['priority'] = 0
+
+            pokemon['dist'] = distance(
+                self.bot.position[0],
+                self.bot.position[1],
+                pokemon['latitude'],
+                pokemon['longitude'],
+            )
+
+            if pokemon['dist'] > self.config['max_distance'] and not self.config['snipe']:
+                continue
+
+            pokemon_list.append(pokemon)
+
+        return pokemon_list
+
+    def addCaught(self, pokemon):
+        for caught_pokemon in self.caught:
+            if caught_pokemon['encounter_id'] == pokemon['encounter_id']:
+                return
+        if len(self.caught) >= 200:
+            self.caught.pop(0)
+        self.caught.append(pokemon)
+
+    def wasCaught(self, pokemon):
+        for caught_pokemon in self.caught:
+            if pokemon['encounter_id'] == caught_pokemon['encounter_id']:
+                return True
+        return False
+
+    def addSeen(self, pokemon):
+        for seen_pokemon in self.seen:
+            if seen_pokemon['encounter_id'] == pokemon['encounter_id']:
+                return
+        if len(self.seen) >= 200:
+            self.seen.pop(0)
+        self.seen.append(pokemon)
+
+    def removeSeen(self, pokemon):
+        for idx in xrange(len(self.seen)):
+            if self.seen[idx]['encounter_id'] == pokemon['encounter_id']:
+                del self.seen[idx]
+                return
+
+    def update_map_location(self):
+        try:
+            r = requests.get('{}/loc'.format(self.config['address']))
+        except:
+            logger.log('Could not reach PokemonGo-Map Server', color='red')
+            return
+        j = r.json()
+
+        dist = distance(
+            self.bot.position[0],
+            self.bot.position[1],
+            j['lat'],
+            j['lng']
+        )
+
+        # update map when 500m away from center and last update longer than 2 minutes away
+        now = int(time.time())
+        if dist > 500 and now - self.last_map_update > 2 * 60:
+            requests.post('{}/next_loc?lat={}&lon={}'.format(self.config['address'], self.bot.position[0], self.bot.position[1]))
```

Comment:
Can we not change the location of PokemonGo-Map if we are snipping? Or add another option to stop this from happening?